host/sgx/calls.c

```diff
@@ -742,6 +763,13 @@ oe_result_t oe_ecall(
     if (!(tcs = _assign_tcs(enclave)))
         OE_RAISE(OE_OUT_OF_THREADS);
 
+    oe_log(
+        OE_LOG_LEVEL_VERBOSE,
+        "%s 0x%x OE_ECALL: %s\n",
```

Comment:
We probably want to be consistent with the other PR which changes this line for OCALLs

Reply:
Done.